exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusUnitsHelper.java

```diff
@@ -69,9 +69,7 @@ private static void initUnit(String otelName, String pluralName, String singular
 
   @Nullable
   static Unit convertUnit(String otelUnit) {
-    if (otelUnit.isEmpty() || otelUnit.equals("1")) {
-      // The spec says "1" should be translated to "ratio", but this is not implemented in the Java
-      // SDK.
+    if (otelUnit.isEmpty()) {
```

Comment:
The reason why I converted OTel's unit `1` to an empty unit in Prometheus is that the previous implementation did that as well https://github.com/open-telemetry/opentelemetry-java/blob/ffd53c7d56f3d14cdb13e8489d18d3d14b71da33/exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusUnitsHelper.java#L175-L176. I'm wondering if anyone has more context on this?

Reply:
The previous implementation only converted "1" to "ratio" for gauges - https://github.com/open-telemetry/opentelemetry-java/blob/ffd53c7d56f3d14cdb13e8489d18d3d14b71da33/exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusMetricNameMapperTest.java#L64

The reason for this was to make sure that it behaved similar to [collector's implementation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c3b2997563106e11d39f66eec629fde25dce2bdd/pkg/translator/prometheus/normalize_name.go#L149).

Reply:
> The previous implementation only converted "1" to "ratio" for gauges -
> The reason for this was to make sure that it behaved similar to [collector's implementation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c3b2997563106e11d39f66eec629fde25dce2bdd/pkg/translator/prometheus/normalize_name.go#L149).

Ah I was trying to understand why it did this, since I couldn't find any language in the spec indicating a special case for ratio and gauge.

I think in this case we should side with the spec over the collector. Special casing `1` -> `ratio` just for gauges is surprising and doing the conversion doesn't seem to cause problems for other types.